sklearn/svm/classes.py

```diff
@@ -575,7 +582,7 @@ class SVC(BaseSVC):
 
     _impl = 'c_svc'
 
-    def __init__(self, C=1.0, kernel='rbf', degree=3, gamma='auto',
+    def __init__(self, C=1.0, kernel='rbf', degree=3, gamma='auto_default',
```

Comment:
Hm I think I liked ``auto_deprecated`` better? bikeshedding a bit, though.

Reply:
Na, in some sense `auto_deprecated` is better, since it gives a clear indication to change back this variable's name in the future.

Do I need to document it in the docstring?

Reply:
It certainly can't hurt